leptos/src/hydration/mod.rs

```diff
@@ -135,15 +135,11 @@ pub fn HydrationScripts(
         .unwrap_or_default();
 
     let root = root.unwrap_or_default();
+    // Workaround for view macro issue with None::<String> syntax
+    let crossorigin_none: Option<String> = None;
```

Comment:
This introduces a breaking change/regression by simply never setting the `crossorigin` attribute in any case.